### ISSUE TITLE
mpd.lan kann kein MPD mehr

### ISSUE THREAD

**Issue by phjlipp** [openlab-aux/orgafoo#245]
Entweder das MPD muss neu eingerichtet werden oder das device muss heruntergenommen werder. Ich würde ersteres bevorzugen.

**Comment by sternenseemann** [comment]
`  mdp.openlab.lan`   geht auch nicht?

**Comment by Profpatsch** [comment]
Seit Streaming geht, machen wir nicht mehr wirklich mpd, weil das immer ein Geficke war. Also bei mir ist es gaaaanz weit unten auf der Prioliste.

**Comment by sternenseemann** [comment]
Cool wäre etwas sshfs-ähnliches für ftp.openlab.lan. Samba wäre cool, ist aber unter non-Windows frickelig.

**Comment by Profpatsch** [comment]
> etwas sshfs-ähnliches

sshfs? :,D

    mount -tnfs -onolock ftp.openlab.lan/data/upload/music /mnt

**Comment by sternenseemann** [comment]
Richtig, auf GNU/Linux ist eigentlich nichts ein Problem, wir müssen aber mal die Anlage Windows- und macOS-kompatibel machen.

**Comment by Profpatsch** [comment]
> wir müssen aber mal die Anlage Windows- und macOS-kompatibel machen.

at

**Comment by sternenseemann** [comment]
Es gibt auch Leute, die nicht PulseAudio benutzen oder Lennart nicht mögen.

**Comment by Profpatsch** [comment]
Viel Erfolg.

**Comment by sternenseemann** [comment]
Bitter.

**Comment by richi235** [comment]
NFS ist das Dateisystem der Zukunft, seit 30 Jahren

**Comment by momo-aux** [comment]
Verstehe das Problem nicht, samba einrichten und fertig. Wenn sich keiner traut mach ich das halt.
Wobei Windows user auch wie alle FTP verwenden können. Geht ja sogar im Explorer.
Sonst halt winscp verwenden. WebDAV wäre auch eine Alternative.

**Comment by Profpatsch** [comment]
Update: mpd ist jetzt wieder vorbereitet und wird vermutlich laufen, sobald ich das nächste Mal im Lab bin. Das nfs-Automounting hat ein bisschen länger gebraucht.

**Comment by phjlipp** [comment]
Mal bitte überprüfen. Wie ich das lese, sollte es funktioneren, tut es aber nicht. Auch nett wäre, wenn mpd.lan wieder ginge, die Adresse ist man schließlich gewohnt.

**Comment by Profpatsch** [comment]
> Wie ich das lese, sollte es funktioneren, tut es aber nicht.

mpd ist in jeder Hinsicht kaputt, und ich habe keinen Bock mehr, da noch mehr Zeit zu investieren. Wenn jemand das unbedingt möchte, kann ich ihm das zeigen und derjenige kann sich dann daran probieren.

> wenn mpd.lan wieder ginge

Die Adresse ist `hannswurscht.openlab.lan`, `mpd.lan` war ein Artefakt von früher, das auch gar nicht in das Namensschema passt. Sollte es auch noch geben, ist halt irgendein odroid vom Privi.

**Comment by muesli** [comment]
JFI: Nach der Netzwerk Umstellung ist mpd jetzt: `mpd.lab`
